report/main.qmd

```diff
@@ -1 +1,166 @@
+---
+title: Comparing intervention effects on zero-calorie beverages consumption
+author: Yian Lin, Haibo Wu, Wakeel Adekunle Kasali
+date: 2024-03-01
+format: 
+  pdf :
+    fontsize: 11pt
+number-sections: true
+header-includes: 
+  \usepackage{caption}
+  \newcommand{\beginAppendix}{
+    \setcounter{table}{0}  
+    \renewcommand{\thetable}{A\arabic{table}}}
+bibliography: ../references/hw2.bib
+---
 
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE, message=FALSE, warning=FALSE)
+```
+
+```{r}
+library(tidyverse)
+#devtools::install_github("stat545ubc-2023/rmiss")
+library(rmiss)
+library(knitr)
+library(here)
+```
+
+## Introduction
+Sugary beverage consumption poses health concerns because of its association with several health challenges like obesity and diabetes. Different intervention methods are proposed to steer consumers toward zero-calorie beverages that are thought healthier. This study aims to investigate the efficacy of five different intervention methods, three of which focus on visual calorie messaging via posters, and the other two focus on price discount. To assess and compare different intervention methods, sales of zero-calorie beverages and sugary beverages under different interventions were monitored in three different sites. 
```

Comment:
Instead of "thought healthier" , "believed to be healthier"
Change to "while the other two" instead of "and the other two"